0.preprocess-sites/8.visualize-plate-layout.py

```diff
@@ -0,0 +1,392 @@
+import os
+import sys
+import pathlib
+import argparse
+import pandas as pd
+import matplotlib.pyplot as plt
+import seaborn as sns
+
+# import numpy as np
+
+sys.path.append(os.path.join("..", "scripts"))
+from config_utils import process_config_file
+from cell_quality_utils import CellQuality
+
+parser = argparse.ArgumentParser()
+parser.add_argument(
+    "--config_file",
+    help="configuration yaml file for preprocessing pipeline",
+    default="site_processing_config.yaml",
+)
+args = parser.parse_args()
+config_file = args.config_file
+
+config = process_config_file(config_file)
+
+# Defines the sections of the config file
+core_args = config["core"]
+spots_args = config["process-spots"]
+summ_cell_args = config["summarize-cells"]
+summ_plate_args = config["summarize-plate"]
+
+# Defines the variables set in the config file
+batch = core_args["batch"]
+quality_func = core_args["categorize_cell_quality"]
+ignore_files = core_args["ignore_files"]
+acquisition_side = core_args["acquisition_side"]
```

Comment:
one very minor thing is that I didn't understand what this was until line 67. Maybe I still don't even understand 😂 

Is this simply how many sites per side of a square grid? I wonder if we could improve the variable name slightly

Reply:
yes, it is just number of sites on the side of the square grid. we can change it :)